### DISCUSSION TITLE
请教：加载预训练模型报错

### DISCUSSION THREAD

**Discussion by firewolf189** [fishaudio/Bert-VITS2#301] [Q&A] [answered]
INFO:models:Loaded checkpoint 'data/output/models/DUR_0.pth' (iteration 0)
ERROR:models:emb_g.weight is not in the checkpoint
INFO:models:Loaded checkpoint 'data/output/models/G_0.pth' (iteration 0)
PytorchStreamReader failed reading zip archive: failed finding central directory
加载预训练模型报错，用的底模是Bert-VITS2_2.3底模_0.0.1

**Comment by Stardust-minus** [comment, marked as answer]
See #279

**Comment by guiji101** [comment]
https://huggingface.co/OedoSoldier/Bert-VITS2-2.3/tree/main this one works for me
